apps/obsidian/tsconfig.json

```diff
@@ -12,5 +12,9 @@
     "importHelpers": true,
     "isolatedModules": true,
     "strictNullChecks": true,
+    "paths": {
```

Comment:
For the record: changes to these files were necessary at some point, but not anymore. Could revert. But It think declaring types explicitly is useful.